packit_service/worker/events/pagure.py

```diff
@@ -228,6 +228,15 @@ def get_base_project(self) -> GitProject:
         logger.debug(f"Base project: {fork} owned by {self.base_repo_owner}")
         return fork
 
+    def get_packages_config(self) -> Optional[PackageConfig]:
```

Comment:
:eyes: Do we have both `project` × `get_project` and `packages_config` × `get_packages_config`? /o\

Reply:
yes I fear :D